axelor-account/src/main/java/com/axelor/apps/account/service/InvoiceVisibilityServiceImpl.java

```diff
@@ -189,4 +196,12 @@ protected boolean _getInvoiceTermsCondition2(Invoice invoice) {
   protected boolean _getDecisionDateCondition(Invoice invoice) {
     return invoice.getDecisionPfpTakenDate() != null;
   }
+
+  protected boolean _getStatusPaymentVouchers(Invoice invoice) {
+    AppAccountService appAccount = Beans.get(AppAccountService.class);
+    return (invoice.getOperationTypeSelect() == InvoiceRepository.OPERATION_TYPE_CLIENT_SALE
+            || invoice.getOperationTypeSelect() == InvoiceRepository.OPERATION_TYPE_SUPPLIER_REFUND)
+        ? appAccount.getAppAccount().getPaymentVouchersOnCustomerInvoice()
+        : appAccount.getAppAccount().getPaymentVouchersOnSupplierInvoice();
```

Comment:
Should be .OPERATION_TYPE_CLIENT_SALE || .OPERATION_TYPE_CLIENT_REFUND

Reply:
We already have a method for that : 
https://github.com/axelor/axelor-open-suite/blob/master/axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceToolService.java#L92

Reply:
But in ticket it's different.

It's
Supplier Invoice or customer refund
Customer Invoice or supplier refund